terraform-plans/templates/github/tox.ini.tftpl

```diff
@@ -22,9 +22,12 @@ commands =
     black --check --diff --color .
     isort --check --diff --color .
     flake8
+%{ if enable_pylint == "yes" ~}
     pylint --recursive=y .
-    # mypy temporarily disabled because it doesn't gracefully recursively find files like the others
-    #mypy --install-types --non-interactive .
+%{ endif ~}
+%{ if enable_mypy == "yes" ~}
+    mypy --install-types --non-interactive .
```

Comment:
It's checking for "yes" here, but in the variables it's setting "true" or "false"

Reply:
Ah, I forget change it to true. Thanks for find it out.